scripts/generate/templates/react.template.tsx

```diff
@@ -0,0 +1,72 @@
+/**
+ * Extend Shopify Checkout with a custom Post Purchase user experience
+ * This Shopify Checkout template provides two extension points:
+ *  1. Inquiry - Called first, during the payment step.
+ *  2. Render - If requested by `Inquiry`, will be rendered after
+ *     payment processing completes
+ */
+
+import React from 'react';
+import {
+  extend,
+  renderReact,
+  InputForRenderExtension,
+  Text,
+} from '@shopify/argo-checkout';
+
+/** Define any shape or type of data */
+interface Payload {
+  couldBe: 'anything' | 'everything';
+}
+
+/**
+ * Entry point for the Inquiry Extension Point.
+ *
+ * Returns a value indicating whether or not to render a PostPurchase step,
+ * and optionally allows data to be stored on the client for use in the
+ * `::Render` extension point.
+ */
+extend('Checkout::PostPurchase::Inquiry', async ({storage}) => {
+  const {render, payload} = await doInquiry();
+  if (render) {
```

Comment:
I think this block warrants a specific call out as being the part doing the actual saving